### PR TITLE
v0.12.0

### PR DESCRIPTION
**Pull request by richardeoin**

### CODE REVIEW

**Comment by richardeoin** [comment]
Any comments / last items before the release are welcome!

@burrbull @mattico @ryan-summers

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,35 +2,49 @@
...
-* timer: `counter()` method on LP Timers returns u16
+* Replace crate::Never with core::convert::Infallible [#339]
+* Added some [defmt](https://defmt.ferrous-systems.com/) support [#304]
+* Add support for FDCAN peripheral using the
+  [fdcan](https://github.com/stm32-rs/fdcan) crate [#277]
+* Add `inner()` and `inner_mut()` methods to more peripherals [#337]
+* pwr: Add backup domain voltage regulator control [#303]
+* pwr: More support for VOS0, and support for VOS2 and VOS3 [#316] [#327]
+* rcc: Add logging (`debug` level) behind `log` feature gate [#318]
+* systick: Impleement
```
Comment (ryan-summers) [inline CHANGELOG.md:29]:
```suggestion
* systick: Implement
```

**Review by ryan-summers — COMMENTED** [review]
LGTM outside of one typo I see - our usage of the HAL hasn't identified any further issues that should be incorporated into the next release :)

**Comment by mattico** [comment]
I've got nothing to add. Thanks Richard!

**Comment by jordens** [comment]
Wohoo! Thanks to all (and especially also @richardeoin @burrbull @mattico ): lots of good things in this release!

**Pull request merged**